drivers/shared/include/vic_driver_shared.h

```diff
@@ -61,6 +61,38 @@ typedef struct {
     size_t N_TYPES[2];
 } param_set_struct;
 
+/******************************************************************************
+ * @brief   This structure stores output information for one output file.
+ *****************************************************************************/
+typedef struct {
+    char prefix[20];         /**< prefix of the file name, e.g. "fluxes" */
```

Comment:
If we're updating it anyway I would some system-wide string size like `MAXSTRING`